vault.js

```diff
@@ -1,4 +1,27 @@
 'use strict';
-module.exports = function() {
-  
-};
\ No newline at end of file
+function vaultGen(){
+  var secretValue = {};
+
+  function getValue (key){
+    if (!key){
+      return null;
+    }else if(!secretValue.hasOwnProperty(key)){
+      return null;
+    }else{
+      return secretValue[key];
+    }
+  }
+
+  function setValue (key, value){
+    secretValue[key] = value;
+    return secretValue;
+  }
+
+return {
```

Comment:
code cleanup. The `return` is inside the vaultGen function and should be indented.
The closing `}` should align with `return`